app/src/lib/design-system-v2/helpers/theme.css

```diff
@@ -80,5 +80,32 @@
   --requestly-color-white: #ffffff;
   --requestly-color-white-t-20: #ffffff33;
   --requestly-color-black: #0e0e0e;
-  --requestly-color-black-t-70: 000000b2;
+  --requestly-color-black-t-70: #000000b2;
```

Comment:
What does  `t` represents in ` --requestly-color-black-t-70`

Reply:
Transparency